src/Pages/Category.jsx

```diff
@@ -1,7 +1,21 @@
-import { React} from 'react';
+import { React, useState, useEffect} from 'react';
+import axios from 'axios';
+import Key from "../config.keys";
 const Category = () => {
-
-
+    const [data, setData] = useState([]);
+    useEffect(() => {
+        window.scrollTo(0, 0);
+        const getEvents = async () => {
+            try {
+                const res = await axios.get( `${Key.BASE_API}/events/categories`);
+                setData({ ...res.data.data });
+            } catch (error) {
+                console.log(error);
+            }
+        };
+        getEvents();
+    }, []);
```

Comment:
Hey! Nice work. Please handle memory leak too since we are making async call. And set a error variable in local state when error occurs.

Reply:
okk